docs/api/scala/infer.md

```diff
@@ -0,0 +1,39 @@
+# Infer API
+The MXNet Scala Infer API provides you with model loading and inference functionality using the MXNet Scala package.
+
+
+## Prerequisites
+To use the Infer API you must first install the MXNet Scala package. Instructions for this are provided with a [tutorial for setting up a project in the IntelliJ IDE](http://mxnet.incubator.apache.org/tutorials/scala/mxnet_scala_on_intellij.html), however you may use your IDE of choice.
```

Comment:
Are all the changes to that tutorial merged? I can't see them.
Also, can we maybe point it to scala installation page rather than taking them to intellij ide setup? Not everyone wants to run code from IDE.
Maybe we can add both links:
1) scala installation page
2) ide setup

What do u think?

Reply:
My instructions in the tutorial are better than the installation page(s). This would require more links since each OS is on a different page.